components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -52,6 +53,12 @@ const (
 	BEBPublishEndpointForPublisher  = "/sap/ems/v1/events"
 
 	reconcilerName = "backend-reconciler"
+
+	certificateSecretNamespace = "kyma-system"
+	certificateSecretName      = "eventing-webhook-server-cert"
```

Comment:
this is made configurable in the helm chart [here](https://github.com/raypinto/kyma/blob/d27c23ff666af5987b9ffde300620f8c9f4222c0/resources/eventing/charts/controller/values.yaml#L51)

Reply:
oh ok.  so I need to read the value from the helm chart?

Reply:
lets discuss this one

Reply:
this is not so critical, hence I leave you the decision to you.

Reply:
I've done it!